packages/ethereum-contracts/contracts/tokens/SETH.sol

```diff
@@ -38,7 +43,12 @@ interface ISETHCustom {
  * @author Superfluid
  */
 // solhint-disable-next-line no-empty-blocks
-interface ISETH is ISETHCustom, ISuperToken { }
+interface ISETH is ISETHCustom, ISuperToken {
+    function getUnderlyingToken() external override(ISETHCustom, ISuperToken) view returns(address tokenAddr);
+    function upgrade(uint256 amount) external override(ISETHCustom, ISuperToken);
+    function upgradeTo(address to, uint256 amount, bytes calldata data) external override(ISETHCustom, ISuperToken);
+    function downgrade(uint256 amount) external override(ISETHCustom, ISuperToken);
```

Comment:
Is there a reason for defining this here since it's already part of the `ISuperToken` interface?

Reply:
because SETHProxy only inherits the ISETHCustom

```
contract SETHProxy is ISETHCustom, CustomSuperTokenProxyBase {
```